torch/csrc/jit/passes/quantization/insert_quant_dequant.cpp

```diff
@@ -339,6 +351,46 @@ void RemoveRedundantQuantizationOps(std::shared_ptr<Graph>& graph) {
   rewriter.runOnGraph(graph, filter);
 }
 
+void ReplicateClampScalarArgs(std::shared_ptr<Graph>& graph) {
```

Comment:
What are you replicating here? Why do you need to replicate the min/max args?

Reply:
this is to ensure these args can be matched in patterns